src/Content/Text/BBCode.php

```diff
@@ -2100,7 +2100,7 @@ public static function getTags($string)
 		$ret = [];
 
 		// Convert hashtag links to hashtags
-		$string = preg_replace('/#\[url\=([^\[\]]*)\](.*?)\[\/url\]/ism', '#$2', $string);
+		$string = preg_replace('/#\[url\=([^\[\]]*)\](.*?)\[\/url\]/ism', '#$2 ', $string);
```

Comment:
Whyyyy?

Reply:
We need a separator. Sometimes the hash links aren't separated. Then there are like `#[url=...]#[url=....`. This function then generated some huge hashtag out of it (like `#news#ufo#alien#landing`) instead of having it separated.

Reply:
And if they are already separated we add an additional space for no reason?

Reply:
Yeah. But this is only done in this function to impre the parsing. So it doesn't matter.